apps/omg_child_chain/lib/omg_child_chain/fee_server.ex

```diff
@@ -88,7 +89,7 @@ defmodule OMG.ChildChain.FeeServer do
 
     with {:reload, changed_at} <- should_load_file(path),
          {:ok, content} <- File.read(path),
-         {:ok, specs} <- Fees.parse_file_content(content) do
+         {:ok, specs} <- FeeParser.parse_file_content(content) do
```

Comment:
may I propose an alternative:

 - keep the parsing functions in the `Fees` module (undo the split)
 - move the `Fees` module entirely into `omg_child_chain`, where I think it belongs

I think it would satisfy your needs, and at the same time keep the fee stuff together (in one module & in one app)

Reply:
after discussion elsewhere:

> move the Fees module entirely into omg_child_chain, where I think it belongs

nope, I'm wrong about this :point_up: 

I see two viable options then:
1/ just revert the change and keep all in omg and revisit later
2/ do what you did in the PR and revisit later
I'd favor 1/ to keep this PR focused and keep all fee stuff together (easier to revisit in the future), but 2/ is also totally fine.